scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3851,418 @@ def rvs(self, dim, size=1, random_state=None):
+        return xx, xcond
+
+    def _checkresult(self, result, cond, bad_value):
+        result = np.asarray(result)
+        if cond.ndim != 0:
+            result[cond] = bad_value
+        elif cond:
+            if result.ndim == 0:
+                return bad_value
+            result[...] = bad_value
+        if result.ndim == 0:
+            return result[()]
+        return result
+
+    def _logpmf(self, x, M, m, n):
+        num = betaln(m+1, 1) - betaln(x+1, m-x+1)
+        den = betaln(M+1, 1) - betaln(n+1, M-n+1)
+        return num.sum(axis=-1) - den
+
+    def logpmf(self, x, m, n):
+        """
+        Log of the Multivariate Hypergeometric probability mass function.
+
+        Parameters
+        ----------
+        x : array_like
+            Quantiles, with the last axis of `x` denoting the components.
+        %(_doc_default_callparams)s
+
+        Returns
+        -------
+        logpmf : ndarray or scalar
+            Log of the probability mass function evaluated at `x`
+
+        Notes
+        -----
+        %(_doc_callparams_note)s
+        """
+        M, m, n, mncond = self._process_parameters(m, n)
+        x, xcond = self._process_quantiles(x, M, m, n)
+
+        result = self._logpmf(x, M, m, n)
+
+        # replace values for which x was out of the domain; broadcast
+        # xcond to the right shape
+        xcond_ = xcond | np.zeros(mncond.shape, dtype=np.bool_)
+        result = self._checkresult(result, xcond_, np.NINF)
+
+        # replace values bad for n or m; broadcast
+        # mncond to the right shape
+        mncond_ = mncond | np.zeros(xcond.shape, dtype=np.bool_)
+        return self._checkresult(result, mncond_, np.NAN)
+
+    def pmf(self, x, m, n):
+        """
+        Multivariate Hypergeometric probability mass function.
+
+        Parameters
+        ----------
+        x : array_like
+            Quantiles, with the last axis of `x` denoting the components.
+        %(_doc_default_callparams)s
+
+        Returns
+        -------
+        pmf : ndarray or scalar
+            Probability density function evaluated at `x`
+
+        Notes
+        -----
+        %(_doc_callparams_note)s
+        """
+        out = np.exp(self.logpmf(x, m, n))
+        return out
+
+    def mean(self, m, n):
+        """
+        Mean of the Multivariate Hypergeometric distribution
+
+        Parameters
+        ----------
+        %(_doc_default_callparams)s
+
+        Returns
+        -------
+        mean : array_like or scalar
+            The mean of the distribution
+        """
+        M, m, n, mncond = self._process_parameters(m, n)
+        M, n = M[..., np.newaxis], n[..., np.newaxis]
+        mu = n*(m/M)
+        if mu.shape[-1] == 1:
+            return self._checkresult(mu, mncond, np.NAN).squeeze(-1)
+        return self._checkresult(mu, mncond, np.NAN)
+
+    def var(self, m, n):
+        """
+        Variance of the Multivariate Hypergeometric distribution.
```

Comment:
![image](https://user-images.githubusercontent.com/6570539/95817126-94443f00-0cd5-11eb-9131-266059187faf.png)

Is it possible you reviewed an old commit? Many of your comments say "outdated", 
![image](https://user-images.githubusercontent.com/6570539/95817185-b6d65800-0cd5-11eb-9b32-5ec9e4821dc4.png)

but @tirthasheshpatel hasn't made a commit since yesterday.